### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by Xenomorphing19**
__21__ 

 

#### Short description of what this resolves:
script.py asks you your email id, password, no. of reciepents, their email ids message and then sends it via command prompt(Terminal)

#### Changes proposed in this pull request and/or Screenshots of changes:

-
-
-

### CODE REVIEW

**Comment by Lokesh2703** [comment]
Check issue number once again and edit PR

**Comment by Xenomorphing19** [comment]
edited sir

**Comment by Lokesh2703** [comment]
Make your code run more efficient with one time login and sending messages to many people.

**Comment by Lokesh2703** [comment]
Why do u close this PR it is perfectly alright just edit login line so that u login only once

**Comment by Lokesh2703** [comment]
I will merge after u edit

**Comment by Xenomorphing19** [comment]
sir ive created a new pr with this issue resolved

**Comment by Xenomorphing19** [comment]
sir i dont know how to edit a pre existing pr

**Comment by deepakDOTexe** [comment]
Edit the file in the branch from which you have created the pull request .
Here the branch is task_3.

**Comment by Xenomorphing19** [comment]
done sir

File: Submissions/Xenomorphing19/Task_3/script.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    li.insert(i,str(s))
+print('Enter the message:')
+mes=raw_input()
+s = smtplib.SMTP('smtp.gmail.com', 587)
+    s.starttls()
+s.login(str(user),str(pwd))
+for i in range(len(li)):  
+    message =str(mes)
+    s.sendmail(str(user),l[i],message) 
+    s.quit()
```
Comment (Lokesh2703) [inline Submissions/Xenomorphing19/Task_3/script.py]: Remove intendation here because due to this mail will be sent to only one person

**Comment by Xenomorphing19** [comment]
Done sir!

**Pull request merged**